bin/update_odoo_param.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# FIXME the below code should be in boot but for an unknown reason, the
+# instruction `grep -q "^$param\s*=" $odoo_conf_file` fails to run in boot, when it
+# works fine here.
+# Both are bash scripts, but:
+#  - boot is run by Docker as an entrypoint
+#  - target_user.sh is run using bash command
+#
+log_src='['${0##*/}']'
+
+odoo_user=$1
```

Comment:
Not relevant since the script is only meant to be called from boot, where the parameters are always correct.